android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -138,8 +145,16 @@ private void checkMandatoryConstraints(MediaConstraints peerConstraints) {
 
     ReadableArray enumerateDevices() {
         WritableArray array = Arguments.createArray();
+
+        ReadableArray uvcDevices = uvcEnumerator.enumerateDevices();
         String[] devices = cameraEnumerator.getDeviceNames();
 
+        if (uvcDevices.size() > 0) {
+            for (int j = 0; j < uvcDevices.size(); j++) {
+                array.pushMap(uvcDevices.getMap(j));
```

Comment:
We need to filter the devices. maybe you can do something like `uvcEnumerator.enumerateDevices(nativeDevices);` where `nativeDevices` is `cameraEnumerator.getDeviceNames();` and you can do the filtering there, if they match.